### PR TITLE
Test backfill 2 electric boogaloo

### PR DESCRIPTION
**Pull request by jimszalew**
LIST OF CHANGES:
-admin login test
-sad path login test
-retired item test

THINGS I'M UNSURE ON:
-non restful actions aplenty in a few controllers, possible chore for refactoring

RSPEC GREEN?
YES

### CODE REVIEW

**Comment by jimszalew** [comment]
I thought it would be a good idea to push my most recent test backfills before diving into my user story. There are still some things that need to be tested, and some controller actions that need to be refactored to follow restful convention.

File: spec/features/items/items_can_be_retired_spec.rb
Comment (mimizotti) [inline spec/features/items/items_can_be_retired_spec.rb:22]: This test looks good.

File: spec/features/users/user_can_login_spec.rb
Comment (mimizotti) [inline spec/features/users/user_can_login_spec.rb:65]: This test looks good as well. Thanks @jimszalew

**Review by mimizotti — COMMENTED** [review]
This pull request looks good to go. Pending a second review. Perhaps @vaidashi?

**Review by vaidashi — APPROVED** [review]
Great, there is still a good amount of refactoring we can eventually do but for now its cool that we're progressing with minimal disasters.

**Pull request merged**
